.devcontainer/devcontainer.json

```diff
@@ -7,7 +7,6 @@
     // List of ports that you want to preview
     // "forwardPorts": [1234, 5678],
     // Script to run to bootstrap the app when the space is created
-    "postCreateCommand": "make all",
```

Comment:
Why is this not needed?

Reply:
Every time we start a codespace it builds all binaries, which is unnecessary and consumes 100% of the CPU of the most basic machine.